internal/common/common_linux.go

```diff
@@ -194,6 +194,17 @@ func VirtualizationWithContext(ctx context.Context) (string, string, error) {
 		}
 	}
 
+	if PathExists(filepath.Join(filename, "1", "environ")) {
+		contents, err := ReadFile(filepath.Join(filename, "1", "environ"))
+
+		if err == nil {
+			if StringsContains([]string{contents}, "container=lxc") {
```

Comment:
`contents` is just a string, so how about to use `strings.Contains` in a standard library?